share/models/fields.py

```diff
@@ -550,3 +555,37 @@ def bulk_related_objects(self):
         # https://github.com/django/django/blob/master/django/db/models/deletion.py#L151
         # Disable django cascading deletes for this field
         raise AttributeError('This is a dirty hack')
+
+
+class EncryptedJSONField(models.BinaryField):
+    """
+    This field transparently encrypts data in the database. It should probably only be used with PG unless
+    the user takes into account the db specific trade-offs with TextFields.
+    """
+    prefix = b'jwe:::'
+
+    def get_db_prep_value(self, input_json, **kwargs):
+        if input_json:
```

Comment:
This is a personal preference, in situations like these it is better practice to short-circuit.
Rather than having this entire method nested in an `if` block, just return early. It makes it easier to follow the logic.

Could you also change `to_python`?